build.gradle

```diff
@@ -163,6 +163,18 @@ spotless {
             return sb.toString()
         }
 
-        bumpThisNumberIfACustomStepChanges(4)
+        //Replace any FQN versions of javax.annotation.Nullable with the jetbrains variant
+        def nullableCheck = Pattern.compile('^\\+.*javax\\.annotation\\.Nullable.*\$', Pattern.UNIX_LINES | Pattern.MULTILINE)
```

Comment:
Is there any case of a mult-iline annotation usage? This would be a lot simpler if it were just replacing `@javax.annotation.Nullable`.

Reply:
The patterns are just ones that spotless defines by default. But in regards to not just doing the same thing as the non patches variant, it is because I don't want it to accidentally replace any cases of vanilla imports in case they happen to appear in a hunk at the top (such as if we are patching in an interface). Which means I need to check that it is a patch addition line.

Reply:
Can't you just check for `@` instead? A vanilla import won't contain it.